.travis.yml

```diff
@@ -2,13 +2,21 @@ services:
   - secure: "ftig8aPk8avtgzOJUHkt5aQm6iwjC1gYtxPBIVO2fnjCXqfes1HzYEKJeygO7v9BTJ1fa5k4BX0hH09waKD23eLq3+U83DlfO8pbM0jVFoO3qc8YwUYF2MDaUIDvjWv05gI5mpHCeorYnp9FjBOKeaSbEbZuxO9p/54xyrFe3tXrBzv3OFYLzbjw+6NH8sXWsLIiF8oBZolLrPp14fpzIC4hd5ogpP3OqHtnKGkm/gDhQct+pl72laBT/GuWpcf0ul/te5WLfo1SJXuGZpi5xCvWL25u/HCGVx1HGwJWtRLK77dsVpPYOnzmz6HnJ990CYu+og4s/5LQzWDvx1TdOMlYy1k5BWRBJyF2cvc91SBl0uK7grkWS4pKSn0QM0zhEc7JtrdwuebflxjinLC1T3o2MWg8bwbtCKyvKTG9ZdVtKKdgY6FdEt91AUlE7DtglwtstAMUnkfeHeQRIHIlLvptsAwoFZxHOh0uiVM3h7hpMl41eElKuFivwi0P2TDcKuNCg4ZCgL1Styryust98e8qBXAttVDgvJf76Hbn7vPWMP59gYjrBvrvng1DYgwXVgS7Pf/JkO1DvDozSk9Ll1Q/yoNXj4WRimXuCzuRb4yb/u4QL3Az7iDdStC/JMyI6as3ZrwsIDMf7JmW7f7XTEpb0YCOdBIEE08mfpP/ByE="
+  - secure: "Rm34yAkJplhsvsd0Wc6kI/0fASwx1jbNX8GQ7EeHU2W+C63i3jtm/yo76YtSqNiL6Ikf0FZNKKWNAz5csc+tE0MmEDDss+QnDvdmBGW02VLplOcruwNwT5lX/iFc8rMqfVtO/06kWQNgi831mggqB7ZfcXeJqxuc7tiXG7gOnm7kbd3OCQcY9xtdoHnxh1Tc55miSUPi/tsF44aM1b/2RjjZA27ECM2HBM9KV5qqPCwQ5Qw6tZ7g7iWyV1V1T7Xeqqx7S8x1CeNx1dDRR24bcm25n06bVhs+XBMz8TdXMXfujAEVilSoEXh0TvAzIr53SCRUUR2WHKKUDukdGPGad9ErcFjZU41gwTlY+X9VH8T3mnVHfA234KF/o5JppdFV2S7xzp8E9CYX9iuhXzbDbJsDh4dgpAB90r9EHrLVzzrETy9BkfRw01OQQsfo8B36Fa+X46JT2nwId+2pe1K93w3MSldAA4/jZcGvIA6UNhf61J2tTHYqU65n37HOGI5o0SBlh4QcmT7D5UAaT/KdOJOAb+hvqON9E3YXpvprzc1cexfdYvq75GP+SNajnL+okWxduy6MhjZcg2AAgJ1/XlsMfs6AJApbTA3TjGlwALAhgBg/ZK4Yk3vGVd4iwUvsSnxNd8NB6Evq5yOqgfcUtw3I71ERIzGQTuIdytqN9bQ="
+  - secure: "SMQR+WxtSViJ5NvERGMLAa3Saend7ReSA2BcPxu9HIWsDILa+J7Iqa2jK40SMzCUH/gOFMpCfvoarKZxPWiHQ1kms9LKTl5dVx6/OcGWTdfUq0G7xncpGxTL2NTygE2gJojXOCUlq0qhoCl3nCBKRBBhCcuiceDq75sBcVCziB50pdPjW5DemnPn22O3wo/aqPA/eadDw8eXm/RXzvdFnIoBKpaaM/3qqloLvBu3uzHpjEdxZ6jLEcvgJyoJHZq/LlnkbEM36+1pF8yZXtoPjYnWZOwT5lxollw2w6ipIQkwsLiMA4F7UufsId3hZWKslybft6GXDMF1ayZ7oU/nP7mS5YyChlWlxflVEhNvmQXSw8CxZ7Haiz5Mn2HOo/hhRG6j43VeLkcM7hH6xkskfYbIuRCCEi6AGK2dCczz4jke91iyYkXLHC8/fCWLTOhQAgE8/ifeh97koP1gs0V6VeLogdbKvyUMbV1ATc3DaPHHWerNOHj/K1A7kRLJAY9+SPPUle1haErlaLVTmvbmyZP60JBTEl+Pv+I9ynhPyap+L0XNq2gXREtkjAzGRbdbTn1XYNOzdnqcgi461czZFhfSgI6b4bvkX++kXsWwu66SMw66/Fr9ayphbzHT1yCD2ZAG+NKYEPthY43p0+BlVB8keZ8t0g15Kjhda2njh1s="
 script:
-- docker build --target test --tag todo-app:test .
-- docker run -e API_KEY=$API_KEY -e TOKEN=$TOKEN -e FLASK_APP=$FLASK_APP -e FLASK_ENV=$FLASK_ENV todo-app:test test_e2e.py
-- docker run todo-app:test test_app.py
-- docker run todo-app:test test_indexViewModel.py
+  - docker build --target test --tag todo-app:test .
+  - docker run -e API_KEY=$API_KEY -e TOKEN=$TOKEN -e FLASK_APP=$FLASK_APP -e FLASK_ENV=$FLASK_ENV todo-app:test test_e2e.py
+  - docker run todo-app:test test_app.py
+  - docker run todo-app:test test_indexViewModel.py

```

Comment:
It would be nicer to group tests in folders so that this list of commands won't grow whenever you add a new test file.